peachjam/js/components/Annotations/AnnotationList.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div>
+    <annotation-item
+      v-for="annotation in items"
+      :key="annotation.id"
+      ref="gutter-item"
+      :annotation-data="annotation"
+      :view-root="viewRoot"
+      :gutter="gutter"
+      :use-selectors="useSelectors"
+    />
+  </div>
+</template>
+<script>
+import AnnotationItem from './Annotation.vue';
+export default {
+  name: 'AnnotationsList',
+  components: {
+    AnnotationItem
+  },
+  props: {
+    annotations: {
+      type: Array,
+      default: () => []
+    },
+    viewRoot: HTMLElement,
+    gutter: HTMLElement,
+    useSelectors: Boolean
+  },
+  data: () => ({
+    items: []
+  }),
+  mounted () {
+    this.getAnnotations();
+  },
+  methods: {
+    getAnnotations () {
+      fetch(`/api/annotations/?document=${this.viewRoot.dataset.documentId}`)
+        .then((resp) => {
+          if (!resp.ok) {
+            throw new Error('Failed to fetch annotations');
+          }
+          return resp.json();
+        })
+        .then((annotations) => {
+          this.items = annotations.results;})
+        .catch((err) => {
+          console.error(err);
+        });
+    },
```

Comment:
rather than loading the annoations from the document detail view, we can fetch them directly from the api endpoint, which solves the caching problem